Podcast/UserDetailSectionHeaderView.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ProfileSectionHeaderView.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 11/16/16.
+//  Copyright © 2016 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+enum UserDetailSectionHeaderType: String {
+    case subscriptions = "Public Subscriptions"
+    case recasts = "Recasts"
```

Comment:
i think it should be "your subscriptions / your recasts" here
![image](https://user-images.githubusercontent.com/20246620/37233565-ae37bac2-23c1-11e8-8e17-0814b4e3a12f.png)

Reply:
But for other people...? This is meant to be a representation of your profile to others no?

Reply:
When the user is viewing their own external profile it should say "Subscriptions" / "Your Recasts"  .. else "Subscriptions" / "\(firstName) Recasts"

Reply:
if someone else is viewing it what you have right now makes sense, but if you're looking at your own profile it would make more sense to make it "yours"